src/graphql/schema/withOperators.ts

```diff
@@ -135,32 +224,73 @@ const defaults: DefaultsType = {
 
 const listOperators = ['in', 'not_in', 'all'];
 
+const gqlTypeCache = new Map();
+
+const getGqlTypeFromCache = (cacheKey: string): GraphQLType => {
+  return gqlTypeCache.get(cacheKey);
+};
+
+const storeGqlTypeInCache = (cacheKey: string, type: GraphQLType) => {
+  gqlTypeCache.set(cacheKey, type);
+};
```

Comment:
I don't understand the need for the map with get and set functions instead of just using a plain old object.